lib/browser/api/power-monitor.js

```diff
@@ -5,4 +5,16 @@ const {powerMonitor, PowerMonitor} = process.atomBinding('power_monitor')
 Object.setPrototypeOf(PowerMonitor.prototype, EventEmitter.prototype)
 EventEmitter.call(powerMonitor)
 
+powerMonitor.on('newListener', (event) => {
+  if (event === 'shutdown' && powerMonitor.listenerCount('shutdown') === 0) {
```

Comment:
Is the new "shutdown" listener added only _after_ this function is called?

Reply:
Yes